flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
aarch64 is not available?

Reply:
I'm trying to build everything from source now and this looks good so far: we build libvips and dependencies fine, but when I try to run npm install inside the sandbox it fails with ENOENT for `/run/build/tropy/package.json`. If I drop in to the build with `--build-shell` and try to run `npm install` myself it runs for a while then fails with EAI_AGAIN. Do you know why that is?